### PR TITLE
Update label text on link shortener checkbox - Issue #470'

### LINKED ISSUE #470: Update label text for link shortener
Related to #468, I updated the back end to use an internal link shortener instead of bit.ly, so the text just needs to be updated to "Shorten Link" instead of "Shorten with bit.ly"

### PR DESCRIPTION
**Pull request by ayushkumarbhadani**
Fixes #470  
Removed Bit.ly shorten checkbox from UI of index.php

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -1102,8 +1102,6 @@
 								<span id="helioviewer-url-box-msg"></span>
 								<form style="margin-top: 5px; text-align: center;">
 									<input aria-label="Viewport URL" type="text" class="helioviewer-url-input-box helioviewer-url-input-box-menu" style="width:98%;" value="https://helioviewer.org" />
-									<label for="helioviewer-url-shorten-1">Shorten with bit.ly? </label>
-									<input id="helioviewer-url-shorten-1" type="checkbox" class="helioviewer-url-shorten" />
```
Comment (dgarciabriseno) [inline index.php:1106]: Actually, can you add this back and just change the text from "Shorten with bit.ly" to just "Shorten link"

File: index.php [hunk 2]
```diff
@@ -1397,8 +1395,6 @@
 				<span id="helioviewer-url-box-msg"></span>
 				<form style="margin-top: 5px;">
 					<input aria-label="Viewport URL" type="text" class="helioviewer-url-input-box helioviewer-url-input-box-video" style="width:98%;" value="https://helioviewer.org" />
-					<label for="helioviewer-url-shorten">Shorten with bit.ly? </label>
-					<input id="helioviewer-url-shorten" type="checkbox" class="helioviewer-url-shorten" />
```
Comment (dgarciabriseno) [inline index.php:1401]: Can you add this back and just change the text from "Shorten with bit.ly" to just "Shorten link"

**Review by dgarciabriseno — CHANGES_REQUESTED** [review]
Hi! Thanks for your contribution!

I actually changed the link shortener we're using and forgot to update this issue. Instead of removing this, can you update the label text?

File: index.php (hunk 1 above)
Comment (ayushkumarbhadani) [reply index.php:1106]: Sure!

Comment (ayushkumarbhadani) [reply index.php:1106]: Hey, I have created new PR here: https://github.com/Helioviewer-Project/helioviewer.org/pull/471

**Pull request closed without merging**